labelbox/schema/export_task.py

```diff
@@ -467,7 +467,8 @@ class ExportTask:
     class ExportTaskException(Exception):
         """Raised when the task is not ready yet."""
 
-    def __init__(self, task: Task) -> None:
+    def __init__(self, task: Task, is_export_v2: bool = False) -> None:
+        self._is_export_v2 = is_export_v2
```

Comment:
I assume we need to explicitly pass it in because of the api feature flag that might change streamable to export_v2?